K2Field.K2NE.ServiceBroker/K2NEServiceBroker.cs

```diff
@@ -242,6 +246,8 @@ public override string GetConfigSection()
             Service.ServiceConfiguration.Add(Constants.ConfigurationProperties.AdditionalADProps, false, ""); //checked
             Service.ServiceConfiguration.Add(Constants.ConfigurationProperties.ADOSMOQueries, false, ""); //checked
             Service.ServiceConfiguration.Add(Constants.ConfigurationProperties.ADOSMOQueriesFile, false, ""); //checked
+            Service.ServiceConfiguration.Add(Constants.ConfigurationProperties.AllowPowershellScript, true, "true"); //checked
+            Service.ServiceConfiguration.Add(Constants.ConfigurationProperties.PowerShellSubdirectories, false, ""); //checked
```

Comment:
Emm... Am I correct that you mean to set a default value for PowerShellSubdirectories property? If yes so done.